relays/tasmota/manifest.yml

```diff
@@ -0,0 +1,85 @@
+# Manifest for blueprint connection to any Tasmota device
+
+blueprint_spec: device/1.0
+
+display_name: Tasmota device
+description: A wrapper around the Tasmota interface to connect to any Tasmota powered device.
+icon: light-switch
+vendor: Tasmota
+author: Hermanos Energy
+support:
+  url: https://go.enapter.com/enapter-blueprint-support
+  email: support@enapter.com
+contributors:
+  - Jan Ruijtenberg
```

Comment:
Just in case – if you add your Github username here (`SoloJan`) instead, it will be shown as a clickable link to your Github profile in the web version.